wiki/Tournaments/MWC/2020_4K/zh.md

```diff
@@ -98,28 +98,65 @@ osu!mania 2020 4K 世界杯由 [osu! team](/wiki/People/The_Team) 和多位社
 
 ---
 
-## 比赛时间表：四分之一决赛
+## 比赛时间表：半决赛
 
-### 2020 年 8 月 29 日，星期六
+### 2020 年 9 月 5 日，星期六
 
-| A 队 |  |  | B 队 | 比赛时间 | A 队当地时间 | B 队当地时间 |
-| --: | --: | :-- | :-- | :-: | :-: | :-: |
-| 荷兰 | ![][flag_NL] | ![][flag_TH] | 泰国 | **20:30 UTC+08** | 14:30 UTC+02 | 19:30 UTC+07 |
-| 巴西 | ![][flag_BR] | ![][flag_GB] | 英国 | **23:30 UTC+08** | 12:30 UTC-03 | 16:30 UTC+01 |
+| A 队 |  |  | B 队 | 比赛时间 | A 队当地时间 | B 队当地时间 |  |
+| --: | --: | :-- | :-- | :-: | :-: | :-: | :-: |
+| 马来西亚 | ![][flag_MY] | ![][flag_TH] | 泰国 | **18:00 UTC+08** | 18:00 UTC+08 | 17:00 UTC+07 | ² |
```

Comment:
Looking at this again, I believe that the match time is supposed to be in UTC.

Reply:
Do u meaning I convert the time zone from +00 to +08? I want Chinese people to view the schedule in their time zone, I also did the same for the previous updates

Reply:
Alright